extensions/wikia/AdEngine/js/video/vastUrlBuilder.js

```diff
@@ -68,3 +72,12 @@ define('ext.wikia.adEngine.video.vastBuilder', [
 		build: build
 	};
 });
+
+// TODO: ADEN-4128 - remove me
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)